source/vocab/relations.ttl

```diff
@@ -444,6 +444,13 @@ prefix relsubtype: <http://id.loc.gov/vocabulary/preservation/relationshipSubTyp
     owl:inverseOf :hasSeries;
     owl:equivalentProperty bf2:seriesOf .
 
+:isIssueOf a owl:ObjectProperty;
+    # :category :pending ?
```

Comment:
I'd rather we allow this for "everybody", if it is what we will recommend for the newspaper data.